nodes/widgets/ui_button.html

```diff
@@ -36,7 +36,26 @@
                 topicType: { value: 'msg' },
                 buttonColor: { value: '' },
                 textColor: { value: '' },
-                iconColor: { value: '' }
+                iconColor: { value: '' },
+                
+                // pointerup/down event support
+                enablePointerdown: { value: false },
+                pointerdownPayload: { value: '', validate: (hasProperty(RED.validators, 'typedInput') ? RED.validators.typedInput('pointerdownPayloadType') : function (v) { return true }) },
+                pointerdownPayloadType: { value: 'str' },
+                enablePointerup: { value: false },
+                pointerupPayload: { value: '', validate: (hasProperty(RED.validators, 'typedInput') ? RED.validators.typedInput('pointerupPayloadType') : function (v) { return true }) },
+                pointerupPayloadType: { value: 'str' },
+
+                iconColor: { value: '' },
+                
+                // pointerup/down event support
+                enablePointerdown: { value: false },
+                pointerdownPayload: { value: '', validate: (hasProperty(RED.validators, 'typedInput') ? RED.validators.typedInput('pointerdownPayloadType') : function (v) { return true }) },
+                pointerdownPayloadType: { value: 'str' },
+                enablePointerup: { value: false },
+                pointerupPayload: { value: '', validate: (hasProperty(RED.validators, 'typedInput') ? RED.validators.typedInput('pointerupPayloadType') : function (v) { return true }) },
+                pointerupPayloadType: { value: 'str' },
+
```

Comment:
```suggestion
                iconColor: { value: '' }
```

duplicate code

Reply:
Oh thanks that has to have happened when merging somathing ... i am fairly unexperienced as you can see.

I just used the name because it is refering to the actual browser event nothing more nothing less but the user would propably be more used to calling it button down and up. So i will rename them.

I thought disabling the events early as possible would be the best or maybe even disabling the eventhandler inside the component in order for the system to not have to compute on something tht gets dropped when it tries to send.